test/functional/feature_logging.py

```diff
@@ -54,6 +58,17 @@ def run_test(self):
         self.start_node(0, ["-debuglogfile=%s" % (invalidname)])
         assert os.path.isfile(os.path.join(invdir, "foo.log"))
 
+        # check that -nodebuglogfile disables logging
+        self.stop_node(0)
+        os.unlink(default_log_path)
+        assert not os.path.isfile(default_log_path)
+        self.start_node(0, ["-nodebuglogfile"])
+        assert not os.path.isfile(default_log_path)
+
+        # just sanity check no crash here
+        self.stop_node(0)
+        self.start_node(0, ["-debuglogfile=/dev/null"])
```

Comment:
Will this work on non-linux platforms? Is `os.devnull` better? https://docs.python.org/3/library/os.html#os.devnull